### DISCUSSION TITLE
Shopify for databases ? [Sep 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6517] [Discord Threads]
> 🙋‍♂️ **imadbws**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### I'd come across this requset before. Could you expand more on the idea - like what are some of the features and flow of your ideal customer.   
####    
#### Like a customer of yours logs in and browses some part of the data with various filter and sort. And then upgrades to browse all. We do 've the option of API and csv which allow the customer to download the data ? I believe that is something you probably dont want or will be added as premium 🙂 So hence more details on the flow and expectations will help.  
#  
> 🙋‍♂️ **imadbws**  
#### YES EXAACTLY into the point  
#  
> 🙋‍♂️ **imadbws**  
#### This some great examples of the project or the feature I want to implement  :    
####    
#### https://findthatlead.com   
#### https://www.lusha.com   
#### https://www.clodura.ai   
#### https://www.apollo.io   
#### https://www.myleadfox.com   
#### https://useartemis.co    
#### https://www.leadfuze.com    
####    
#### Anyways the commun idea between all of them is to have already data ( leads in excel including emails, names, jobs ect..) stored in server or admin dashboard then the users can start filtre,search and download these leads depends on their credits or a monthly plan.    
####    
#### Very important import feature that should be implemented in as it  will helps alot of lead suppliers and agencies like us to deliver leads to their clients in a secure and professional way.  
#  
> 🙋‍♂️ **imadbws**  
#### Just a simple and basic version of the examples I sent you can do the job all what matters is to restrict users for access & download the database unless the credits they paid for .  
#  
> 🙋‍♂️ **imadbws**  
#### if you already received this kind of request or have already the solution implemented will be very happy to light me up how I should use it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1142834763439292567)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
